saleor/graphql/account/mutations/account.py

```diff
@@ -0,0 +1,220 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....account import emails, events as account_events, models, utils
+from ....checkout import AddressType
+from ...account.enums import AddressTypeEnum
+from ...account.types import Address, AddressInput, User
+from ...core.mutations import BaseMutation, ModelDeleteMutation, ModelMutation
+from .base import (
+    INVALID_TOKEN,
+    BaseAddressDelete,
+    BaseAddressUpdate,
+    BaseCustomerCreate,
+)
+
+
+class AccountRegisterInput(graphene.InputObjectType):
+    email = graphene.String(description="The email address of the user.", required=True)
+    password = graphene.String(description="Password", required=True)
+
+
+class AccountRegister(ModelMutation):
+    class Arguments:
+        input = AccountRegisterInput(
+            description="Fields required to create a user.", required=True
+        )
+
+    class Meta:
+        description = "Register a new user."
+        exclude = ["password"]
+        model = models.User
+
+    @classmethod
+    def save(cls, info, user, cleaned_input):
+        password = cleaned_input["password"]
+        user.set_password(password)
+        user.save()
```

Comment:
```suggestion
        user.save(update_fields=['password'])
```

Reply:
We can't use update_fields for an object without primary key.